dev/archery/archery/tests/test_bot.py

```diff
@@ -156,6 +156,7 @@ def handler(command, **kwargs):
 
 @pytest.mark.parametrize(('command', 'reaction'), [
     ('@ursabot build', '+1'),
+    ('@ursabot build\nwith a comment', '+1'),
```

Comment:
Thanks Jon for working on this!

Could you please add a test case with additional content preceding the mention?

Reply:
Absolutely, I wasn't 100% certain this was the appropriate place to test this — so if it's not let me know.

Also, I expect that test to fail (and respond a -1), though we could probably add support for it if we wanted.

Reply:
Turns out that it is a no-op, just like other examples where the `@` mention is not first.

Reply:
@jonkeane thanks for adding a test case!

We should probably add support for `@` mention anywhere in the comment by splitting the lines first.
Could you please create a follow-up jira for this use case (or directly implement it :) in this PR)?

Reply:
SGTM